src/register/EmbeddableRegistrationPage.jsx

```diff
@@ -303,16 +374,26 @@ const EmbeddableRegistrationPage = (props) => {
     Object.keys(configurableFormFields).forEach((fieldName) => {
       if (fieldName === 'country') {
         payload[fieldName] = configurableFormFields[fieldName].countryCode;
+        validatedFormFields[fieldName] = configurableFormFields[fieldName].countryCode;
       } else {
         payload[fieldName] = configurableFormFields[fieldName];
       }
     });
-
     // Don't send the marketing email opt-in value if the flag is turned off
     if (!flags.showMarketingEmailOptInCheckbox) {
       delete payload.marketingEmailsOptIn;
     }
-
+    let isDirty = false;
+    const dirtyFormFields = {};
```

Comment:
We don't need this `dirtyFormFields` dict to save invalidated fields, In `validateInput` function error state is already setting and that can be used here.

Reply:
addressed

Reply:
Could i please know why are we still keeping `dirtyFormFields` and not using errors?

Reply:
addressed